css-fonts-and-colors/css-fonts-and-colors-quiz-notes.md

```diff
@@ -0,0 +1,40 @@
+# css-fonts-and-colors-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- Name two types of units that can be used to adjust `font-size` in CSS.
+  font-weight and font-style
```

Comment:
Not quite! It's asking what we can use to only control the `font-size` property. `font-weight` & `font-style` are also properties